js/DisableUiComponent.js

```diff
@@ -70,7 +70,9 @@ DisableUiComponent.prototype.enable = function (target) {
             .removeClass('js-disabled u-cursor-not-allowed');
 
         // Remove wrapper which provides visually disabled styling
-        $this.unwrap('<div class="u-ui-disabled"></div>');
+        if ($this.parent().hasClass('u-ui-disabled')) {
```

Comment:
Do we need the check? I thought `unwrap()` failed silently?

> If an element's parent does not match the selector, the element won't be unwrapped.

Reply:
This is needed as `.unwrap([selector])` is not supported in the version of jQuery currently in use.